src/test/java/com/wikia/webdriver/pageobjectsfactory/componentobject/ad/AutoplayVuap.java

```diff
@@ -202,6 +229,10 @@ private void waitFor(final Predicate<AutoplayVuap> predicate, final long timeout
         .until(predicate);
   }
 
+  private boolean isOverlayNoVisible() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "isOverlayNoVisible" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)